systemtests/tests/bareos-concurrency/testrunner-client-concurrency

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+set -o pipefail
+set -u
+#
+# Run a simple backup
+#   then restore it.
+#
```

Comment:
Could you add a description to the tests that matches what they are doing ?  Its not clear to me what exactly  we are trying to test here.